src/EventService.Models.Db/DbUserBirthday.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace LT.DigitalOffice.EventService.Models.Db;
+
+public class DbUserBirthday
+{
+  public const string TableName = "UsersBirthdays";
+
+  [Key]

```

Comment:
is it required? mb just PK in migration?

Reply:
Yes